deno.jsonc

```diff
@@ -104,6 +104,7 @@
     "merkle-reference": "npm:merkle-reference@^2.0.1",
     "multiformats": "npm:multiformats@^13.3.2",
     "react": "npm:react@^18.3.1",
+    "react-dom": "npm:react-dom@^18.3.1",
```

Comment:
This isn't ideal, will mean this may be included in the compiled artifact, but I understand why its here